CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,5 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v69.0.0...main)
+
+- The bundled version of Glean has been updated to v34.0.0.
```

Comment:
Yeah OK, this was an unrelated changes, but I added it because I was messing around with changelogs already...